src/main/java/blusunrize/immersiveengineering/common/blocks/plant/HempBlock.java

```diff
@@ -107,22 +110,35 @@ public EnumPlantType getPlantType(IBlockReader world, BlockPos pos)
 	}
 
 	@Override
-	public VoxelShape getShape(BlockState state, IBlockReader worldIn, BlockPos pos)
+	@SuppressWarnings("deprecation")
+	public VoxelShape getShape(BlockState state, IBlockReader worldIn, BlockPos pos, ISelectionContext context)
 	{
 		return shapes.getOrDefault(state.get(GROWTH), VoxelShapes.fullCube());
 	}
 
+	//TODO: Could not see a proper way to access a fortune values using the builder (e.g. luck scaling is -1024..1024)
+	//      -> added a concept loot table file data/immersiveengineering/loot_tables/blocks/hemp.json.
+	// 				 for experimenting with alternatives to overriding `getDrops()`.
 	@Override
-	public void getDrops(BlockState state, NonNullList<ItemStack> drops, World world, BlockPos pos, int fortune)
+	public List<ItemStack> getDrops(BlockState state, LootContext.Builder builder)
 	{
+		final int fortune = 0;
```

Comment:
Can't you get the fortune value from the loot context?
Edit: Isn't this whole method unnecessary now that you've written the loot table JSON?

Reply:
I did not find out how. Traced the functionality down to variable/variant property parsing from JSONs. There is a luck value that is a builder input. When you actually want to build you have to specify a loot param set, which as checked required and optional properties. This maps to types, conditions, etc. It actually throws if something is not right (or I am missing an easy bypass like "default empty param set" or so.
So that was the reason why I concept JSON file. It does not have exactly the drop probabilities as the 1.12 method. If this is okay, then we probably should go for the json file.

Reply:
This looks like it should work (and does compile):
```java
ItemStack tool = builder.get(LootParameters.TOOL);
int looting = 0;
if(tool!=null)
	looting = EnchantmentHelper.getEnchantmentLevel(Enchantments.FORTUNE, tool);
```
Most of the parameters seem to be set before `getDrops` is called, (hopefully) including `LootParameters.TOOL`.

Reply:
Ok, I would actually prefer this solution over a loot table JSON file, so let's drop the JSON.